### ISSUE TITLE
Ascertainment of direction of effect

### ISSUE THREAD

**Issue by Daniel-Considine** [opentargets/issues#3282]
## Background

Raised from discussion on slack genetics channel with @Juanmaria-rr. 

https://platform.opentargets.org/target/ENSG00000105397/associations
https://genetics.opentargets.org/variant/19_10364976_C_A

If we look at this variant for TYK2 & lymphocyte count, we see "loss of function | risk". There are a few problems with how this has been assigned: 

1. Splice-QTL beta coefficients shouldn't be used for direction of effect in this way, as it isn't informative enough alone. For example, a splice-QTL with a negative coefficient can be increasing gene expression/function, by increasing the quantity of non-canonical isoforms which are still biologically functional. Juan has said this is a known issue. 

2. The platform is taking gene effect direction from the  QTL with the largest absolute beta coefficient. This is biased towards tissue/QTL studies with lower power, hence why the above sQTL (4.3E-7) is used instead of very strong eQTL signals in the opposite direction (6.4e-168). Yakov and I think using the effect direction of the QTL with the smallest P-value is more sensible. Possibly some form of weighting across QTL studies could also be used. 

3. More of a general issue, but many quantitative traits probably shouldn't be assessed as protective vs risk. In the above example, generally **lower** genetically determined lymphocyte count will be associated with **increased risk** of disease. Others can be more complex, where being at either extreme confers disease risk, such as body height and weight.

**Comment by Juanmaria-rr** [comment]
I created an issue for the work in progress to solve the sQTL effect size assessment when creating the column that takes the highest effect size, which address the point 1. 

Regarding the point 3, I think is an interesting one. I agree that If we were able to robustly connect the measurements with diseases, this would enrich these type of evidences. As far as I know there is no current information enough from this type of evidence to give another step from the measurement and scoping whether it is related with a disease in a given direction **in a systematic way**. Maybe in the GWAs there are some metadata about the trait that can help to scope this (?). It may sound that risk/protective should refer to a disease or something particularly relevant to the individual/population. 

[This](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9012149/) could be a nice example where we could connect it in the way that @Daniel-Considine said, but IMO this represent more an hypothesis building, where one could argue that the given variant could be associated with increased AD, which is not true, unless you have evidences linking the variant with AD.

**Comment by d0choa** [comment]
1- sQTL are on the way to be removed as discussed
2- We could use lowest p-value in the context of (1). @Juanmaria-rr is there an easy way for you to check the effects in your analysis? I will amend #3283
3- This has come out several times. We need a systematic solution if we want it to apply to behavioural and quantitative traits. The only thing that occurs to me is that we could generalise the wording but nothing obvious comes to mind. Any ideas @Daniel-Considine?

**Comment by Daniel-Considine** [comment]
> This has come out several times. We need a systematic solution if we want it to apply to behavioural and quantitative traits. The only thing that occurs to me is that we could generalise the wording but nothing obvious comes to mind. Any ideas @Daniel-Considine?

Yeah it's difficult to think of terminology that works for everything. Dichotomous traits/diseases should be fine as they are, but maybe something like concordant/discordant could work for quantitative traits. @addramir any thoughts?
